packages/ui/src/multiplying-architecture/KaotoEditorApp.tsx

```diff
@@ -23,7 +24,7 @@ export class KaotoEditorApp implements Editor {
   constructor(
     private readonly envelopeContext: KogitoEditorEnvelopeContextType<KaotoEditorChannelApi>,
     private readonly initArgs: EditorInitArgs,
-    private readonly catalogUrl: string,
+    private readonly settingsAdapter: AbstractSettingsAdapter,
```

Comment:
@apupier, here's how to use it, so instead of passing the `catalogUrl` directly, we can pass the `SettingsAdapter` so we propagate the entire configuration object.